src/com/projectkorra/projectkorra/firebending/combustion/Combustion.java

```diff
@@ -103,8 +103,7 @@ public static boolean removeAroundPoint(final Location loc, final double radius)
 	}
 
 	private void advanceLocation() {
-		ParticleEffect.FIREWORKS_SPARK.display(this.location, 5, Math.random() / 2, Math.random() / 2, Math.random() / 2, 0);
-		ParticleEffect.FLAME.display(this.location, 2, Math.random() / 2, Math.random() / 2, Math.random() / 2);
+		ParticleEffect.FIREWORKS_SPARK.display(this.location, 8, Math.random() / 2, Math.random() / 2, Math.random() / 2, 0);

```

Comment:
Maybe we could spice this up a tad to throw in another nice aesthetic change?

Reply:
I'll message my idea to you on slack. I wasn't sure whether it'd be approved so I didn't do it.